crates_io_tarball/src/lib.rs

```diff
@@ -87,7 +88,7 @@ pub fn process_tarball<R: Read>(
             let mut contents = String::new();
             entry.read_to_string(&mut contents)?;
             vcs_info = CargoVcsInfo::from_contents(&contents).ok();
-        } else if entry_path == manifest_path {
```

Comment:
Technically, this means that the order we load `Cargo.toml` and `cargo.toml` may be undefined if both are present in a tarball. I don't think this is worth actually doing anything about (it's like an edge case of an edge case at this point), but it does feel worth calling out.

Reply:
I had actually thought about that, but figured it's currently not worth addressing it. I have some more plans with this package though, so in the future I might fix it :)